### ISSUE TITLE
[BUG] (Text doesn't fit in buttons after installation)

### ISSUE THREAD

**Issue by john-preston** [martinet101/ElevenClock#486]
## Describe the bug
I've installed the version 3.0.0 from GitHub Releases and I got small buttons cutting text after installation (see the screenshot).

The buttons look small and the left one cuts the text. This is Windows 11 in Parallels Desktop on macOS, but I think this happens in a non VM as well. 

## To Reproduce
Steps to reproduce the behaviour:
1. Go to https://github.com/martinet101/ElevenClock/releases/tag/3.0.0
2. Download and launch ElevenClock.Installer.exe
3. Finish the installation

## Expected behaviour
The buttons should be larger and not cutting the text.

## Screenshots
![image](https://user-images.githubusercontent.com/17900494/149616898-5b489da0-1c21-4b15-bc66-9cb56bb9702b.png)

Here is how the screen resolution settings look like:

![image](https://user-images.githubusercontent.com/17900494/149616929-a9d680f1-b2f9-45e2-b468-e076ff24e334.png)
![image](https://user-images.githubusercontent.com/17900494/149616941-b896e325-f6a3-4be4-9bc2-859b1b222ce8.png)

## The Log

```
🔴 Traceback (most recent call last):

🔴   File "tools.py", line 114, in setSettingsValue

🔴   File "globals.py", line 11, in loadTimeFormat

🔴 NotImplementedError: loadTimeFormat function has not been defined!

🔴 Note this traceback was caught by reporter and has been added to the log (loadTimeFormat function has not been defined!)
---------------------------------------------------------------------------------------------------

   ElevenClock's 3.0.0 (v3) log: Select all the text and hit Ctrl+C to copy it
   All modules loaded successfully and sys.stdout patched correctly, starting main script
   Translator function set language to "en"

---------------------------------------------------------------------------------------------------

 Log legend:
 🔵: Verbose
 🟢: Information
 🟡: Expected warning
 🟠: Unexpected warning
 🔴: Error

🟡 This is the primary screen and is set to be skipped
🟢 Loaded clocks in 0.8360302448272705
🔵 Restart loop: 1
🔵 System tray icon enabled
🔵 Starting update check
🟡 This is the primary screen and is set to be skipped
🔵 Restart loop: 1
🟡 This is the primary screen and is set to be skipped
🟡 This is the primary screen and is set to be skipped
🔵 Initializing clock 0...
🔵 Using taskbar background color
🟢 Regular sized taskbar
🔵 Monitor geometry: PySide2.QtCore.QRect(0, 0, 3456, 2160)
🟡 Taskbar at bottom
🟢 Clock on the right
🟢 Using qt's default positioning system
🔵 Clock geometry: PySide2.QtCore.QRect(3056, 2064, 400, 96)
🔵 Font families: ['Segoe UI Variable Display', 'sans-serif']
🔵 Font size: 9.3
🟢 Using white text (dark mode)
🔵 Full screen rect:  (0, 0, 3456, 2160)
🔵 Show/hide loop started with parameters: HideonFS:True, NotHideOnTB:False, HideOnRDP:False, ClockOn1Mon:True, NefWSMethod:True
🟢 Default settings loaded
🟡 Closing clock on {'Monitor': (0, 0, 3456, 2160), 'Work': (0, 0, 3456, 2064), 'Flags': 1, 'Device': '\\\\.\\DISPLAY1'}
🔵 Initializing clock 0...
🔵 Using taskbar background color
🟢 Regular sized taskbar
🔵 Monitor geometry: PySide2.QtCore.QRect(0, 0, 3456, 2160)
🟡 Taskbar at bottom
🟢 Clock on the right
🟢 Using qt's default positioning system
🔵 Clock geometry: PySide2.QtCore.QRect(3056, 2064, 400, 96)
🔵 Font families: ['Segoe UI Variable Display', 'sans-serif']
🔵 Font size: 9.3
🟢 Using white text (dark mode)
🔵 Full screen rect:  (0, 0, 3456, 2160)
🔵 Show/hide loop started with parameters: HideonFS:True, NotHideOnTB:False, HideOnRDP:False, ClockOn1Mon:True, NefWSMethod:True
🟢 Loaded everything in 2.0838139057159424
🔵 Restart loop: 4
🔵 Restart loop: 3
🔵 Version URL: https://versions.somepythonthings.tk/versions/elevenclock.ver
🟢 Updates not found
0 1
PySide2.QtCore.QRect(89, 32, 32, 32)
PySide2.QtCore.QRect(89, 32, 32, 32)
PySide2.QtCore.QRect(147, 32, 32, 32)
🔵 Restart loop: 2
🔵 Restart loop: 1
🔵 Starting update check
🔵 Version URL: https://versions.somepythonthings.tk/versions/elevenclock.ver
🟢 Updates not found
```

**Comment by martinet101** [comment]
Oh, gonna fix that

**Comment by martinet101** [comment]
Fixed, Coming with version 3.1.0

![image](https://user-images.githubusercontent.com/53119851/149625567-3d4ab821-01e6-4656-a8a8-6ae485519337.png)
